packages/server/utils/AzureDevOpsServerManager.ts

```diff
@@ -301,6 +322,75 @@ class AzureDevOpsServerManager {
     return json
   }
 
+  async createTask({
+    rawContentStr,
+    integrationRepoId
+  }: {
+    rawContentStr: string
+    integrationRepoId: string
+  }): Promise<CreateTaskResponse> {
+    const {title} = splitDraftContent(rawContentStr)
+    const {error, projects} = await this.getAllUserProjects()
```

Comment:
Feels like heavy operation to just create a task. We only need `instanceId` here. Is it possible to get it another way? Maybe it should be part of `integrationRepoId` which we just `split`? 

For example  in jira:

`const {cloudId, projectKey} = JiraProjectId.split(integrationRepoId)`